### DISCUSSION TITLE
OCR support for images. Image to text.

### DISCUSSION THREAD

**Discussion by philippeowagner** [GitJournal/GitJournal#756] [Feature Requests]
I'm currently using the [Notebloc app](https://play.google.com/store/apps/details?id=com.notebloc.app) to backup my hand written journal. 

In Notebloc it's possible to switch between the scan / image and edit the text extracted using OCR. 

I'm migrating from a hand written journal to a hybrid approach (plain text and hand written notes). For this I'm switching back to GitJournal  and would like to have OCR support for images within the app.

I tried https://github.com/victorqribeiro/ocr – works great. Maybe this could be integrated easily.

---
> Please vote on this issue with a 👍 reaction. It's one of the primary mechanisms used to prioritize issues. Please also subscribe to the issue if you're interested, as that way I can discuss possible implementations and share mockups.

**Comment by vHanda** [comment]
This actually seems do-able. Gather the votes, and I'll make this a priority :)
